### DISCUSSION TITLE
Official list of all approved image and video formats for displaying on OnlyM

### DISCUSSION THREAD

**Discussion by bvalvik** [AntonyCorbett/OnlyM#490] [General]
We have many individuals who come to us with various formats of images and we find they fail some times.  The latest was obvious as it was a PowerPoint file.  To keep from asking if this willl work, or will that work, do you have an official list of image formats and video formats that will work with OnlyM.  I would like to write up a document to all our presenters for future use if they submit files to be displayed during their parts.

Thank you,

Bill

**Comment by AntonyCorbett** [comment]
Hi Bill. Thanks, that would be a useful document. Here’s the relevant code:

OnlyM.Core/Services/Media/MediaProviderService.cs

Kind regards
Antony

**Comment by geostanko** [comment]
where can I find that document/data ? we had the same problem recently

**Comment by AntonyCorbett** [comment]
@bvalvik offered to create the list based on the code cited above
